robot-server/tests/integration/http_api/protocols/test_deck_coordinate_load.tavern.yaml

```
@@ -0,0 +1,49 @@
+test_name: Upload, analyze, and confirm correctness of loading via deck coordinate names
+
+marks:
+  - usefixtures:
+      - run_server
+      - clean_server_state
+stages:
+  - name: Upload deck_coordinate_load protocol
+    request:
+      url: '{host:s}:{port:d}/protocols'
+      method: POST
+      files:
+        files: 'tests/integration/protocols/deck_coordinate_load.py'
+    response:
+      save:
+        json:
+          protocol_id: data.id
+          analysis_id: data.analysisSummaries[0].id
+      status_code: 201
+      json:
+        data:
+          id: !anystr
+          protocolType: python
+          files:
+            - name: deck_coordinate_load.py
+              role: main
+          createdAt: !re_search "^\\d{4}-\\d{2}-\\d{2}T\\d{2}:\\d{2}:\\d{2}\\.\\d+\\+\\d{2}:\\d{2}$"
+          robotType: OT-2 Standard
+          metadata:
+            protocolName: Deck Coordinate PAPIv2 Test
+          analyses: []
+          analysisSummaries:
+            - id: !anystr
+              status: pending
+  - name: Retry until analyses status is completed and result is ok.
+    max_retries: 5
+    delay_after: 1
```

```suggestion
    max_retries: 10
    delay_after: 0.1
```

Most of our existing tests (with one exception) seem to use these polling parameters. The tighter `delay_after` seems beneficial to me. Imagine if every one of these Tavern tests took a full second.